lib/LaTeXML/Package/xparse.sty.ltxml

```diff
@@ -15,6 +15,14 @@ use strict;
 use warnings;
 use LaTeXML::Package;
 
+# Newer versions of xparse *may* want to defer to one of these versioned files
+# They are also acceptable for interpretation, but must avoid the de-versioning hack/fallback.
+AssignValue("INTERPRETABLE_DEFINITIONS_xparse-2018-04-12.sty" => 1);
+AssignValue("INTERPRETABLE_DEFINITIONS_xparse-2020-10-01.sty" => 1);
```

Comment:
The underscores followed by dashes... 

Any chance I can sell you one something like later on?
```perl
PushValue("BLESSED_RAW_FILES", "xparse-2020-10-01.sty");
```

Can wait until a future PR, but I wanted to mark it early to avoid being called inconsistent in my pedantry :>

Reply:
No stripes with checks. Nobody can accuse you of not being consistently pedantic.